packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -1424,10 +1424,20 @@ export class DeltaManager
         }
 
         // Watch the minimum sequence number and be ready to update as needed
-        assert(this.minSequenceNumber <= message.minimumSequenceNumber, "msn moves backwards");
+        if (this.minSequenceNumber > message.minimumSequenceNumber) {
```

Comment:
How about we create assertFluidOpInvariant() or some other name that is less verbose and achieves same thing?
There is no need to add clientID - it's already in telemetry and easy to find

Reply:
(Much more than one might suspect) I really appreciate you offering up a name @vladsud !  Where would be a reasonable place to `export` it (if it is not file local)?  Would packages/utils/telemetry-utils/src/assert.ts (file does not yet exist) make sense?

Reply:
telemetry-utils is generic telemetry package (pretty much not fluid specific), it should not take dependency on protocol or other packages. So I think either driver-utils or container-utils (or maybe new package?)